app/dashboard/src/providers/AuthProvider.tsx

```diff
@@ -133,7 +133,7 @@ function createUsersMeQuery(
   performLogout: () => Promise<void>,
 ) {
   return reactQuery.queryOptions({
-    queryKey: ['usersMe', session?.clientId] as const,
+    queryKey: [remoteBackend.type, 'usersMe', session?.clientId] as const,
```

Comment:
not sure we need backend type here: this query doesn't depend on the `backend`.
Also, it makes invalidation harder, since we no longer can type `client.invalidateQueries({ queryKey: 'usersMe' })

Reply:
fair... what about changing to `'backend', 'usersMe'` for (marginally) better namespacing, and moving the backend type to the third argument (for the *other* queries that do depend on backend type)? that way we can invalidate queries without any extra fancy logic in `backendHooks`

also more importantly, we would be able to invalidate all backend queries at once without touching any potential non-backend queries (e.g. if we want to also handle PM queries in RQ in the future)

Reply:
(update: no longer strictly necessary as of latest code)

Reply:
In the recent PR I mentioned using an object notation for queries, did you read it? I think this notation is better that the Array notation

Reply:
true, definitely a much better idea. but i guess it might (?) be a bit big of a refactor to do in a relatively small PR like this one - you might want to open an issue for it to keep track of it? (and possibly plan it out0

Reply:
Yeah, it's definitely out of the scope.

Issue: https://github.com/enso-org/cloud-v2/issues/1448